inc/cgi.hpp

```diff
@@ -0,0 +1,15 @@
+//
+// Created by Kevin Di nocera on 3/16/23.
+//
+
+#ifndef WEBSERV_CGI_HPP
+#define WEBSERV_CGI_HPP
+#include <fcntl.h>
```

Comment:
A little return line ?
```suggestion
#define WEBSERV_CGI_HPP

#include <fcntl.h>
```

Reply:
done